### PR TITLE
fix(components/molecule/select): accept onblur prop

### LINKED ISSUE #2282: 🪲 Request to add "onBlur" and "onFocus" events to SelectField Field Component
### URL

https://sui-components.vercel.app/workbench/molecule/selectField/demo

### Steps to reproduce

1. Click on the select field
2. Click outside of the options/input

### Expected Behavior

As a developer, I would like to add a callback function to an "onBlur"/"onFocus" event
So that I can give the user feedback on whether this field is required or not

### Screenshots

_No response_

### Which device have you used?

- [X] Desktop
- [ ] Tablet
- [ ] Mobile
- [ ] smartTV

### Which operating systems have you used?

- [ ] Windows
- [ ] Linux
- [X] macOS

### browser and version

- Chrome Versión 104.0.5112.79
- Node 16.15.0
- npm 8.5.5

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by alfdocimo**
ISSUES CLOSED: #2282

## Molecule/Select

**TASK**: 

### Types of changes

- [x] 🪲 Bug fix (non-breaking change which fixes an issue)
- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🧾 Documentation
- [ ] 📷 Demo
- [ ] 🧪 Test
- [ ] 🧠 Refactor
- [ ] 💄 Styles

### Description, Motivation and Context
Select should accept an onBlur prop

### CODE REVIEW

File: components/molecule/select/src/index.js [hunk 1]
```diff
@@ -125,7 +127,10 @@ const MoleculeSelect = forwardRef((props, forwardedRef) => {
     }
   }
 
-  const handleFocusOut = () => setFocus(false)
+  const handleFocusOut = e => {
```
Comment (andresin87) [inline components/molecule/select/src/index.js:130]: plz, be explicit on variable naming:

Comment (andresin87) [inline components/molecule/select/src/index.js]: plz, be explicit on variable naming:
```suggestion
  const handleFocusOut = event => {
```

File: components/molecule/select/src/index.js [hunk 2]
```diff
@@ -125,7 +127,10 @@ const MoleculeSelect = forwardRef((props, forwardedRef) => {
...
+    onBlur(e)
```
Comment (andresin87) [inline components/molecule/select/src/index.js]:
```suggestion
    onBlur(event)
```

File: components/molecule/select/src/index.js (hunk 1 above)
Comment (alfdocimo) [reply components/molecule/select/src/index.js]: good catch, updated!

File: components/molecule/select/src/index.js (hunk 2 above)
Comment (alfdocimo) [reply components/molecule/select/src/index.js]: ty, updated! 🙌

**Review by andresin87 — APPROVED** [review]

**Pull request merged**